m4/ax_package_versioned.m4

```diff
@@ -0,0 +1,30 @@
+# ===========================================================================
+# https://github.com/andreas-bok-sociomantic
+# ===========================================================================
+#
+# SYNOPSIS
+#
+#   AX_PACKAGE_VERSIONED
+#
+# DESCRIPTION
+#
+#   Split the PACKAGE_VERSION into MAJOR, MINOR and PATCH parts and assign to
+#   separate variables
+#
+# LICENSE
+#
+#   Copyright (c) 2017 Brian Aker <andreas.bok@sociomantic.com>
+#
+#   Copying and distribution of this file, with or without modification, are
+#   permitted in any medium without royalty provided the copyright notice
+#   and this notice are preserved. This file is offered as-is, without any
+#   warranty.
+
+#serial 3
+AC_DEFUN([AX_PACKAGE_VERSIONED],
+    [AC_PREREQ([2.63])dnl
+
+    major=`echo $VERSION | cut -f 1 -d .`
+    AC_SUBST([PACKAGE_VERSIONED],["$PACKAGE$major"])
+    ])
```

Comment:
Also do you really need a m4 extension for this? Why not doing it in the `deb/build` script?